### ISSUE TITLE
rt.cpan.org: breakage

### ISSUE THREAD

**Issue by sjn** [uBlockOrigin/uAssets#21780]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://rt.cpan.org/`

### Category

breakage

### Description

CAPTCHA fails because telemetry POST action is blocked.

FF network log gets the following entries:
```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://cfebec326e6e.73685e66.us-east-1.token.awswaf.com/cfebec326e6e/13de985d72d1/8803f912ede9/telemetry. (Reason: CORS request did not succeed). Status code: (null).

Uncaught (in promise) TypeError: NetworkError when attempting to fetch resource.
```

### Other extensions used

LastPass

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.54.0
Firefox: 120
filterset (summary):
 network: 105381
 cosmetic: 44191
 scriptlet: 18548
 html: 1999
listset (total-discarded, last-updated):
 default:
  user-filters: 1-0, never
  ublock-filters: 36977-206, now
  ublock-badware: 7755-11, now
  ublock-privacy: 946-0, now
  ublock-unbreak: 2222-0, now
  ublock-quick-fixes: 203-0, now
  easylist: 76765-608, now
  easyprivacy: 33103-793, now
  urlhaus-1: 10125-0, now
  plowe-0: 3789-1, now
filterset (user): [array of 1 redacted]
trustedset:
 added: [array of 2 redacted]
switchRuleset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 271 ms (selfie)
 maxAssetCacheWait: 77 ms
popupPanel:
 blocked: 1
 no-cosmetic-filtering: true
 network:
  awswaf.com: 1
```
</details>

**Comment by mauke** [comment]
Ah, interesting. A while ago I worked around the issue by adding a custom rule for rt.cpan.org that allows access to awswaf.com:
```
rt.cpan.org awswaf.com * noop
```

**Comment by mauke** [comment]
I removed my custom rule and logged out of my rt.cpan.org account. I then tried to log in again.
=> I landed on a blank page (title was something like "human verification") because I block all 3rd-party scripts, so none of the captcha was loading.

I then allowed all 3rd-party scripts and tried again.
=> I got the captcha described in this ticket, but solving it didn't redirect me back. I was stuck on the solved captcha (due to the issue described in this ticket).

I then added the custom filter given above, `@@||awswaf.com/*/telemetry$domain=cpan.org`, and reloaded the page and solved the captcha again.
=> I got through and was redirected to the usual rt.cpan.org index page. Success!
